### ISSUE TITLE
Increase maximum zoom level of the NVDA Magnifier

### ISSUE THREAD

**Issue by SaschaCowley** [nvaccess/nvda#19486]
### Is your feature request related to a problem? Please describe

Some users of the new NVDA Magnifier may have severely impaired vision. For these users, 10&times; magnification is likely to be insufficient.

### Describe the solution you'd like
Increase the maximum magnification level:
* Windows Magnifier supports up to 16&times; magnification;
* Zoom Text claims to support up to 36&times; magnification;
* My electronic magnifier supports up to 22&times; magnification.

### Describe alternatives you've considered
Force these users to use another magnification package

### Additional context
While some may argue that if you need higher levels of magnification than 10&times; you should be using speech and/or braille instead, this argument is rather unconvincing. For one thing, users may already be using them, but occasionally want to view what is on screen, for example to use an inaccessible GUI, or to inspect pictures.

As far as I can see, there is no documented maximum scale factor for the Windows magnification API. At a minimum, I would suggest a maximum of 16&times; since we know that the API is capable of this level of magnification since Windows Magnifier can do it.

**Comment by CyrilleB79** [comment]
I fully support this request.
We may increase it to 16x and investigate more upon future user request.

**Comment by CyrilleB79** [comment]
Note that Zoomtext can zoom until 60% since about 2013; before, the max zoom level was 36x. I guess, if they implemented such a high level, that it's because one can use very large screens. In any case, there seem to be a use case for such high levels.

I am not saying that we should implement such high levels, because Zoomtext probably has more efficient smooting algorithms at high levels. But we should at least support the same max level as Windows Magnifier does, since we probably use the same APIs.
